content/sensu-core/2.0/reference/events.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Events"
+description: "The events reference guide."
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: true
+menu:
+  sensu-core-2.0:
+    parent: reference
+---
+
+# Events
+
+## Events
+
+Sensu events are created to acknowledge that a check result has been received. Potentially noteworthy events may be processed by one or more event handlers to do things such as send an email, or invoke an automated action. Every Sensu event provides context, called “event data”, which contains information about the originating Sensu client and the corresponding check result.
+
+### How are Sensu events created?
+
+A Sensu Event is created every time a check result is processed by the Sensu server, regardless of the status indicated by the check result. An Event is created by collating data from the check result, the client registry and additional context added at the time of processing.
+
+### Event actions
+
+- **create:** Indicates a check result status change from zero to non-zero.
+- **resolve:** Indicates a check result status change from a non-zero to zero.
+- **flapping:** Indicates a rapid change in check result status.
+
+### Viewing
+
+To view all the events that are currently in history for a configuration, enter:
+
+```sh
```

Comment:
Any shell formatting should be updated to use hugo's formatter:
```
{{< highlight shell >}}
... text ...
{{< /highlight >}}
```

Reply:
Haha yeah I totally saw that in your PR and forgot to ask. Oh, hugo! Thanks!